third_party/move/move-compiler-v2/tests/ability-checker/ability_violation.exp

```diff
@@ -398,13 +496,29 @@ fun ability::invalid_copy() {
      var $t2: bool
      var $t3: ability::Impotent
      var $t4: ability::Impotent
+     # abort state before: {returns}
```

Comment:
Consider removing dumping of most pipeline stages (using `dump_for_only_some_stages` in testsuite.rs) in this directory so that the attention can be drawn to the diagnostics (which is the main point of this checker). We can also do this as a follow on PR.